src/pages/Lend/LoanChannelCategoryClimateExperiment.vue

```diff
@@ -0,0 +1,236 @@
+			// return `/lend/${this.$route.params.category || ''}`;
+			return this.lendFilterExpVersion === 'b'
+				? this.getAlgoliaFilterUrl()
+				: `/lend/${this.$route.params.category || ''}`;
+		},
+	},
+	apollo: {
+		preFetch: true,
+		query: ecoCategoryPageQuery
+	},
+	created() {
+		let allChannelsData = {};
+
+		try {
+			allChannelsData = this.apollo.readQuery({
+				query: ecoCategoryPageQuery,
+				variables: { basketId: this.loanQueryVars.basketId }
+			});
+		} catch (e) {
+			logReadQueryError(e, 'LoanChannelCategoryControl created ecoCategoryPageQuery');
+		}
+
+		// Filter routes on param.category to get current path
+		this.targetedLoanChannelURL = this.$route?.params?.category;
+
+		// Isolate targeted loan channel id
+		this.targetedLoanChannelID = getTargetedChannel(this.targetedLoanChannelURL, allChannelsData);
+
+		// Get secondary loan channel id's
+		this.secondaryLoanChannelIds = secondaryEcoLoanChannels
+			.map(secondaryLoanChannelURL => { return getTargetedChannel(secondaryLoanChannelURL, allChannelsData); });
+
+		// Assign our initial loan channel based on the route
+		this.loanChannel = allChannelsData?.lend?.loanChannels?.values
+			.filter(channel => channel.id === this.targetedLoanChannelID)?.[0];
+
+		/*
+		 * Experiment Initializations
+		*/
+
+		// Lend Filter Redirects
+		this.initializeLendFilterRedirects();
+	},
+	mounted() {
+		this.updateLendFilterExp();
+		// check for newly assigned bounceback
+		const redirectFromUiCookie = this.cookieStore.get('redirectFromUi') || '';
+		if (redirectFromUiCookie === 'true') {
+			this.cookieStore.remove('redirectFromUi');
+			this.$router.push(this.getAlgoliaFilterUrl());
+		}
+	},
+	methods: {
+		getAlgoliaFilterUrl() {
+			// get match channel data
+			const matchedUrls = _filter(
+				this.loanChannelQueryMap,
+				channel => {
+					return channel.url === this.$route.params.category;
+				}
+			);
+			// check for fallback url
+			const fallback = _get(matchedUrls, '[0]fallbackUrl');
+			if (typeof fallback !== 'undefined') {
+				return fallback;
+			}
+			// use algolia params if available
+			const algoliaParams = _get(matchedUrls, '[0]algoliaParams') || '';
+			if (algoliaParams !== '') {
+				return `/lend/filter?${algoliaParams}`;
+			}
+			// use default
+			return '/lend/filter';
+		},
+		initializeLendFilterRedirects() {
+			const lendFilterEXP = this.apollo.readFragment({
+				id: 'Experiment:lend_filter_v2',
+				fragment: experimentVersionFragment,
+			}) || {};
+			this.lendFilterExpVersion = lendFilterEXP.version;
+
+			// Update Lend Filter Exp CASH-545
+			this.getLendFilterExpVersion();
+		},
+	},
+	beforeRouteLeave(to, from, next) {
+		if (typeof window !== 'undefined'
+			&& to.path.indexOf('/lend/') !== -1
+			&& to.path.indexOf('/lend/filter') === -1) {
+			// set cookie to signify redirect
+			this.cookieStore.set('redirectFromUi', true);
+		}
+		next();
+	}
+};
+</script>
+
+<style lang="scss" scoped>
```

Comment:
I'm assuming this component is temporary, since there's scss usage, so I'll not comment on this file more.

Reply:
@dyersituations we can probably cleanup this `heading-region` section to match the declarations from the LoanSearchPage header area. And do the same the LoanChannelCategoryControl... that way they would all match

Reply:
These kinds of things are good candidates for small cleanup tickets in Vue queue.

Reply:
This was just legacy CSS from LoanChannelPage, which shares the same heading region